docker/entrypoint.sh

```diff
@@ -36,7 +36,7 @@ EOT
         exec dumb-init "$@"
         ;;
     etcd)
-        exec "$@" -advertise-client-urls "http://$DOCKER_IP:2379"
+        exec "$@" -advertise-client-urls "http://${PATRONI_ETCD_HOST:-$DOCKER_IP:2379}"
```

Comment:
Meh. The `PATRONI_ETCD_HOST` is a Patroni configuration, not Etcd and should not be used in advertise-client-urls